proto/info/edge_node_cluster.proto

```diff
@@ -0,0 +1,236 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.info;
+option go_package = "github.com/lf-edge/eve-api/go/info";
+option java_package = "org.lfedge.eve.info";
+
+import "info/info.proto";
+
+enum KubeNodeStatus {
+  STATUS_UNKNOWN = 0;
+  STATUS_READY = 1;
+  STATUS_NOT_READY = 2;
+  STATUS_SCHEDULING_DISABLED = 3;
+  STATUS_OUT_OF_DISK = 4;
+  STATUS_MEMORY_PRESSURE = 5;
+}
+
+message KubeNodeInfo {
+  // Name of the node
+  string name = 1;
+
+  // Status of the node
+  KubeNodeStatus status = 2;
+
+  // Role of the node is server or not
+  bool roleServer = 3;
+
+  // Age of the node, in seconds ago
+  uint64 age = 4;
```

Comment:
Since there can be a delay between info message published and received by the controller, wouldn't an absolute timestamp be better?

Reply:
Yeah I can change the timestamps to a "google.protobuf.Timestamp" type instead.